x-pack/plugins/apm/e2e/cypress/support/step_definitions/apm.ts

```diff
@@ -7,14 +7,17 @@
 import { Given, When, Then } from 'cypress-cucumber-preprocessor/steps';
 import { loginAndWaitForPage } from '../../integration/helpers';
 
+/** The default time in ms to wait for a Cypress command to complete */
+export const DEFAULT_TIMEOUT = 60 * 1000;
```

Comment:
I think the default timeout is 4 seconds. You can change that here: https://docs.cypress.io/guides/references/configuration.html#Timeouts

Reply:
In fact, I copied this default timeout from another file:
- https://github.com/elastic/kibana/blob/03b6c6d638ee4e3432a793e7d239fe9605ef079b/x-pack/plugins/apm/e2e/cypress/integration/helpers.ts#L13-L14

I guess, if required, it could be potentially refactored in another follow up including the above-mentioned snippet.